consensus/polybft/fsm.go

```diff
@@ -19,7 +18,8 @@ import (
 )
 
 type blockBuilder interface {
-	Reset() error
+	Reset()
```

Comment:
I think we are not using this `Reset` function anywhere but in `blockBuilder` constructor, so maybe remove it from the interface?

Reply:
Fixed cf149bdd68be46cd6c649fa3438ee51abc9f5ca4